part2/Crypotology(Lab1)/src/main/java/lab1/FermatTest.java

```diff
@@ -0,0 +1,48 @@
+package lab1;
+
+import java.math.BigInteger;
+import java.util.Random;
+
+// https://en.wikipedia.org/wiki/Fermat_primality_test
+public class FermatTest
+{
```

Comment:
Please use jUnit for tests

Reply:
It's not a class for testing.
FermatTest class' functionality is tested in PrimesTest.java, but agree that naming is misleading. Will be changed.